namerd/examples/3rdparty.yaml

```diff
@@ -0,0 +1,7 @@
+metadata:
+  name: d-tab.l5d.io # the hyphen is required by the Kubernetes API. This will be converted to the CamelCase name "DTab".
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
+description: stores dtabs used by Buoyant's `namerd` service
+versions:
+  - name: v1
```

Comment:
Since this isn't a namerd config, I think it's a bit confusing to put it in the `namerd/examples` directory.  For instance, we can't run `./sbt namerd-examples/3rdparty:run`.  How about moving it to a namerd/examples/k8s subdirectory, or something similar?